src/EditorFeatures/Test/BraceHighlighting/AbstractBraceHighlightingTests.cs

```diff
@@ -19,6 +20,8 @@ public abstract class AbstractBraceHighlightingTests
         {
             using (var workspace = await CreateWorkspaceAsync(markup))
             {
+                WpfTestCase.RequireWpfFact($"{nameof(AbstractBraceHighlightingTests)}.{nameof(TestBraceHighlightingAsync)} creates asynchronous taggers");
```

Comment:
Having the error actually say the problem is really nice, especially when I'm fixing thousands of tests. :-)